modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+
+=== Images that used a Data URI were corrupted when the data wasn’t Base64 encoded
+
+Previously, the {productname} upload logic incorrectly assumed all data URIs used Base64 encoding. As a consequence, if a data URI used, for example, percent-encoding (also known as URL encoding), the data was corrupted.
+
+For this update, the upload logic has been corrected. Data URIs are no longer assumed to all be Base64 encoded and using other encoding methods (such as percent-encoding) does not cause the data to be corrupted.
+
+
+=== Notifications did not properly reposition when toggling fullscreen mode
+
+A {productname} toolbar option, *Fullscreen*, toggles the {productname} editor between the size it takes by default and, as the toolbar’s name suggests, fullscreen. Fullscreen is the largest size available to the editor (ie, taking up all the space available to {productname} in the browser).
+
+If a {productname} instance had a custom notification, `+editor.notificationManager.open+`, set to display when the *Fullscreen* button was used, toggling from Fullscreen back to default resulted in the notification displaying outside the {productname} editor.
+
+{productname} 6.1 corrects this. When toggling between the two *Fullscreen* states, custom notifications now display immediately below the toolbar in either state, as expected.
+
+
+=== Dialogs no longer exceed window height on smaller screens
+
+When re-sizing the viewport while a {productname} dialog is open, the dialog did not start re-sizing until the viewport boundary begins encroaching on the dialog. As a consequence, the dialog was partially cut off in the re-sized viewport (specifically, the bottom border and several pixels of the dialog’s buttons were over-shot) and it could become difficult to distinguish the dialog from the viewport.
+
+With this update, this behavior has been corrected. When a {productname} dialog is open and the viewport is re-sized, the dialog now re-sizes such that both the top and bottom borders of the dialog are always within the current viewport.
+
+
+=== Some function types in the TreeWalker API missed that it could return `undefined`
+
+Backspacing through material in an active {productname} editor could, in some circumstances, result in the insertion point stepping into a table and continue deleting where the expected behavior was for the insertion point to stop on the line below the table.
+
+In investigating and fixing this issue (see immediately below) it was also found that some function types in the `tinymce.dom.TreeWalker` class did not detect that the class could return an `undefined` value.
+
+This has been corrected and all `tinymce.dom.TreeWalker` functions now correctly detect `undefined` values when these are returned.
+
+
+=== In some cases pressing the Backspace or Delete key would incorrectly step into tables rather than remain outside
```

Comment:
Maybe this should be above the previous release note? It reads weirdly to have the intro section to the above be before this entry.